docs/usage.rst

```diff
@@ -0,0 +1,7 @@
+=====
```

Comment:
Is this unfinished? At least from the file name, the point of this file seems superceded by the contents of README.rst.